go.mod

```diff
@@ -3,11 +3,7 @@ module github.com/stripe/skycfg
 go 1.16
 
 require (
-	github.com/golang/protobuf v1.4.1
-	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
```

Comment:
These are the result of running `go mod tidy`: it appears that we don't use these two deps anywhere.